src/main/java/core/basesyntax/WorkWithFile.java

```
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyTotal = 0;
+        int buyTotal = 0;
 
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(",");
+                if (parts.length == 2) {
+                    String operationType = parts[0].trim();
+                    int amount = Integer.parseInt(parts[1].trim());
+                    if (operationType.equals("supply")) {
+                        supplyTotal += amount;
+                    } else if (operationType.equals("buy")) {
+                        buyTotal += amount;
+                    }
+                }
+            }
+        } catch (IOException | NumberFormatException e) {
+            e.printStackTrace();
```

According to the checklist, you should not ignore exceptions. Instead of using `e.printStackTrace()`, re-throw a `RuntimeException` with a descriptive message and the original exception.

```java
throw new RuntimeException("Can't read data from the file " + fromFileName, e);
```